index.js

```diff
@@ -71,92 +114,97 @@ module.exports = {
             /**
              *  snap - save a screenshot image as PNG to the 'report' directory
              *  @param filename {String} - should be unique within the report directory and indicate which
-             *                test it is associated with
+             *                             test it is associated with
              *  @returns {Promise} - upon successful completion, Promise will resolve to a JSON object as below.
-             *              If Jenkins environment variables are found, imageUrl will be added
-             *              {
-             *                  'imageName': 'myImage.png',
-             *                  'imagePath': '/path/to/image/'
-             *                  [, 'imageUrl': 'jenkinsURL']
-             *              }
+             *                       If Jenkins environment variables are found, Jenkins image URLs will be added
+             *                       {
+             *                           'imageName': 'myImage.png',
+             *                           'imagePath': '/path/to/image/'
+             *                           [, 'imageUrl': 'jenkinsUrl', 'archivedImageUrl': 'jenkinsUrl' ]
+             *                       }
              */
-            'snap': function (filename) {
+            'snap': function(filename) {
                 var deferred = nemo.wd.promise.defer(),
-                    imageName,
-                    imageObj = {'imageName': null, 'imagePath': null};
+                    imageObj = {
+                        'imageName': null,
+                        'imagePath': null
+                    },
```

Comment:
@nikulkarni You are making me work very hard for this PR :) 

Looks like my JS beautifier did the re-formatting, and brought out the the diff. Re-formatted anonymous function bracket with a space. Btw, if you guys are open with .editorconfig and some jsbeautifier rules, we could help to standardize.